examples/contiki/server.c

```diff
@@ -215,13 +215,8 @@ PROCESS_THREAD(coap_server_process, ev, data)
   while(1) {
     PROCESS_YIELD();
     if(ev == tcpip_event) {
-      coap_tick_t now;
-
-      coap_ticks(&now);
       /* There is something to read on the endpoint */
-      coap_context->endpoint->sock.flags |= COAP_SOCKET_CAN_READ;
-      /* read in, and send off any responses */
-      coap_read(coap_context, now);  /* read received data */
+      coap_run_once(coap_context, 0);
```

Comment:
PR #347 has introduced pre-processor symbols for the special values that can be passed to `coap_run_once()`. Here, it should be `COAP_RUN_BLOCK` instead of 0.

Reply:
Good catch - missed that following a rebase.  Code now pushed with the correction.